client/src/components/MyMap.js

```diff
@@ -12,6 +12,28 @@ function MyMap() {
 	const [isModalOpen, setModalOpen] = useState(false);
 	const [selectedRhythm, setSelectedRhythm] = useState({});
 
+	function toggleDropdownShown(name) {
+		setRhythmCodes((prevRhythmCodes) => {
+			const newRhythmCodes = [];
+			for (let i = 0; i < prevRhythmCodes.length; i++) {
+				const currentRhythmCode = prevRhythmCodes[i];
+				if (currentRhythmCode.rhythm_code === name) {
+					const updatedRhythmCode = {
+						...currentRhythmCode,
+						isOn: !currentRhythmCode.isOn,
```

Comment:
Given my previous comment, can we continue to use the term `isDropdownShown` instead of `isOn`?

Reply:
name fixed